frontend/src/@types/feedback.ts

```diff
@@ -25,3 +25,11 @@ export interface RevieweeFeedbackData {
   feedbackText: string;
   recommendationPoint: number;
 }
+
+export interface ReviewerFeedbackData {
+  feedbackId: number;
+  receiverId: number;
+  evaluationPoint: number;
+  feedbackKeywords: string[];
+  feedbackText: string;
+}
```

Comment:
요거는 RevieweeFeedbackData 와 거의 동일해서 아래와 같이 쓰는건 어떤가요??

```suggestion
export type ReviewerFeedbackData = Omit<RevieweeFeedbackData, 'recommendationPoint'>;
```

Reply:
아주 좋은 제안이십니다 굳굳